pkg/nvlib/info/api.go

```diff
@@ -23,9 +23,10 @@ type Interface interface {
 
 // Properties provides a set of functions to query capabilities of the system.
 //
-//go:generate moq -stub -out properties_mock.go . Properties
+//go:generate moq -rm -stub -out properties_mock.go . Properties
```

Comment:
what does the `-rm` do? Remove before regenerating?

Reply:
Yes. I added it here, because I always get errors and have to manually remove the file before generating the mocks.